.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "client/packages/plugins/civ-plugins"]
+	path = client/packages/plugins/civ-plugins
+	url = https://github.com/msupply-foundation/civ-plugins.git
+[submodule "client/packages/plugins/example-plugins"]
+	path = client/packages/plugins/example-plugins
+	url = https://github.com/msupply-foundation/open-msupply-plugins.git
+[submodule "client/packages/plugins/civ_plugins"]
+	path = client/packages/plugins/civ_plugins
+	url = https://github.com/msupply-foundation/civ-plugins.git
+[submodule "client/packages/plugins/open-msupply-plugins"]
+	path = client/packages/plugins/open-msupply-plugins
+	url = https://github.com/msupply-foundation/open-msupply-plugins.git
```

Comment:
How do we feel about having submodule references in here? It makes it a lot easier when switching branches, but I can understand it might be confusing for those not working with submodules/plugins.

Reply:
Are they duplicated here? My thought was we wanted to be able to turn them on and off somehow before we're check anything in.

Reply:
Lets remove this file for now